src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -9,14 +9,18 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import junit.framework.Test;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'junit.framework.Test'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)